<ui version="4.0" >
 <class>restoreDialog</class>
 <widget class="QWidget" name="restoreDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="restoreText" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushRestoreOk" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>90</y>
     <width>105</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushRestoreCancel" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>90</y>
     <width>105</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
